### PR TITLE
is/hw1

### PR DESCRIPTION
**Pull request by SapryginIgor**

### CODE REVIEW

**Comment by yana-tanch** [comment]
1) Внутри класса Config не определен gpu_ok, он определен выше, но это нарушение инкапсуляции

2) Для val_loader и test_loader значения параметров shuffle=True, drop_last=True (валидация и тест должны быть воспроизводимы и происходит потеря части данных 
при оценке модели)

3) loss = loss_fn(y_logits.squeeze(), y.to(float))
К какому типу происходит преобразование тензора? 
Лучше сделать явное преобразование y.to(torch.float32) или использовать метод тензора y.float(), который так же приводит к torch.float32

4) При тестировании model.eval, нет вызова метода

5) Нет графиков точности на обучающих и валидационных данных

**Comment by SapryginIgor** [comment]
Пофиксил

> 1. Внутри класса Config не определен gpu_ok, он определен выше, но это нарушение инкапсуляции
> 2. Для val_loader и test_loader значения параметров shuffle=True, drop_last=True (валидация и тест должны быть воспроизводимы и происходит потеря части данных
>    при оценке модели)
> 3. loss = loss_fn(y_logits.squeeze(), y.to(float))
>    К какому типу происходит преобразование тензора?
>    Лучше сделать явное преобразование y.to(torch.float32) или использовать метод тензора y.float(), который так же приводит к torch.float32
> 4. При тестировании model.eval, нет вызова метода
> 5. Нет графиков точности на обучающих и валидационных данных

**Review by yana-tanch — APPROVED** [review]
